openwrt-openwisp-monitoring/files/monitoring.agent

```diff
@@ -31,6 +32,25 @@ fi
 
 URL="$BASE_URL/api/v1/monitoring/device/$UUID/?key=$KEY"
 
+tmp_dir="/tmp/monitoring"
+
+check_available_memory(){
+	total=$(free | grep Mem | awk '{print $2}')
+	available=$(free | grep Mem | awk '{print $7}')
```

Comment:
I take available memory here

Reply:
You could use `ubus call system info` to get the information in one call

```
root@st-dev-07 ~ # ubus call system info
{
        "localtime": 1626076905,
        "uptime": 1443394,
        "load": [
                1760,
                1728,
                64
        ],
        "memory": {
                "total": 4117020672,
                "free": 3917463552,
                "shared": 2035712,
                "buffered": 33484800,
                "available": 3958448128,
                "cached": 62803968
        },
        "swap": {
                "total": 0,
                "free": 0
        }
}
```

Reply:
Done